src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -263,7 +272,7 @@ def fetch_image(url, cache, label):
         try:
             with urlopen_retry(url) as request:  # nosec
                 shutil.copyfileobj(request, image_tempfile)
-        except Exception:  # pylint: disable=broad-except
+        except (urllib.error.URLError, IOError):
```

Comment:
URLError is a subclass of `OSError`, and `IOError` is an alias for `OSError`, so you can simplify to just `except OSError`. `except IOError` works too, but we need to train ourselves to use the new canonical name (myself included, I still write IOError everywhere :grin: )